server/data/migrations/20190301162501_billing.js

```diff
@@ -34,14 +33,16 @@ exports.up = function(knex, Promise) {
       .unsigned();
     billing.string("accountStatus", 128).notNullable();
     billing.integer("targetsUsed").unsigned();
-    
     billing.integer("accountType").unsigned();
     billing.string("mobilePhone", 128);
     billing.string("homePhone", 128);
-    billing.integer('users_id').unsigned();
+    billing
+      .integer('users_id')
+      .unsigned()
+      .references('users.id');
```

Comment:
So using the users.id this will fix the issue with foreign key vs using users_id? Because the `billing.inteneger('users_id')` its just throwing me off

Reply:
it is just referencing the users id. Its identifying it as users (table). id (column). Just setting up a foreign key reference.